components/Users/NewsletterSubscriptions.js

```diff
@@ -132,57 +140,74 @@ class UpdateSubscription extends Component {
-  <Query
-    query={GET_NEWSLETTER_SUBSCRIPTION}
-    variables={{ id: userId }}
-  >
-    {({ data, loading, error }) => {
-      const isInitialLoading =
-        loading && !(data && data.user)
-      return (
-        <Loader
-          loading={isInitialLoading}
-          error={error}
-          render={() => {
-            const { user } = data
-            const {
-              subscriptions,
-              status
-            } = user.newsletterSettings
-            const hasNonEligibleSubscription = subscriptions.some(
-              newsletter => !newsletter.isEligible
-            )
+const NewsletterSubscriptions = ({ t, userId }) => {
+  const [colorScheme] = useColorContext()
 
-            return (
-              <Section>
-                <SectionTitle>
-                  Abonnierte Newsletter
-                </SectionTitle>
-                Status: {status}<br />
-                {hasNonEligibleSubscription &&
-                  'Es können nur User mit aktiver Membership die Republik-Newsletter abonnieren.'}
-                {subscriptions.map(
-                    (subscription, index) => (
-                      <UpdateSubscription
-                        t={t}
-                        key={`${
-                          subscription.name
-                        }-${index}-${
-                          subscription.subscribed
-                        }`}
-                        user={user}
-                        subscription={subscription}
-                      />
-                    )
-                  )}
-              </Section>
-            )
-          }}
-        />
-      )
-    }}
-  </Query>
-)
+  return (
+    <Query query={GET_NEWSLETTER_SUBSCRIPTION} variables={{ id: userId }}>
+      {({ data, loading, error }) => {
+        const isInitialLoading = loading && !(data && data.user)
+        return (
+          <Loader
+            loading={isInitialLoading}
+            error={error}
+            render={() => {
+              const { user } = data
+              const { subscriptions, status } = user.newsletterSettings
+              const hasNonEligibleSubscription = subscriptions.some(
+                newsletter => !newsletter.isEligible
+              )
+
+              return (
+                <Section>
+                  <SectionTitle>Abonnierte Newsletter</SectionTitle>
+                  <div>
+                    <span style={{ marginRight: 10 }}>Status: {status}</span>
+                    {status !== 'subscribed' && (
+                      <Mutation mutation={RESUBSCRIBE_EMAIL}>
+                        {(mutate, { loading, error }) => {
+                          if (error) return <ErrorMessage error={error} />
+
+                          return (
+                            <div>
+                              <TextButton
+                                disabled={loading}
+                                {...colorScheme.set('color', 'primary')}
```

Comment:
See [Avoid Custom Elements](https://styleguide.republik.ch/dev/colors#avoid-custom-elements) in the styleguide color scheme docs.

Does text button really need primary color? If yes could you use a child that is a pre styled `A`? Alternatively do a `<span {...colorScheme.set('color', 'primary')}>`